### PR TITLE
[DOCS] ButtonList - extra mouse buttons

### PR DESCRIPTION
**Pull request by KellyThomas**

### CODE REVIEW

File: doc/classes/@GlobalScope.xml [hunk 1]
```diff
@@ -911,8 +913,10 @@
 			Middle Mouse Button Mask
 		</constant>
 		<constant name="BUTTON_MASK_XBUTTON1" value="128" enum="ButtonList">
+			Extra Mouse Button 1 Mask
 		</constant>
 		<constant name="BUTTON_MASK_XBUTTON2" value="256" enum="ButtonList">
+			Extra Mouse Button 1 Mask
```
Comment (OvermindDL1) [inline doc/classes/@GlobalScope.xml]: `s/1/2/`

Comment (KellyThomas) [reply doc/classes/@GlobalScope.xml]: Thanks!  Always good to have another set of eyes look over things.

**Review by mhilbrunner — COMMENTED** [review]
Not sure whether the first two should be without the `Mask` as the other keys surrounding them don't have it either?

File: doc/classes/@GlobalScope.xml [hunk 2]
```diff
@@ -886,8 +886,10 @@
 			Middle Mouse Button
 		</constant>
 		<constant name="BUTTON_XBUTTON1" value="8" enum="ButtonList">
+			Extra Mouse Button 1 Mask
```
[the remark this answers is not in the document]
Comment (KellyThomas) [reply doc/classes/@GlobalScope.xml]: Right you are, thanks.

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thank you!

**Pull request merged**